main.js

```diff
@@ -34,18 +34,33 @@ var affirmationBTN = document.querySelector("#affirmation");
 var mantraBTN = document.querySelector("#mantra");
 var revieveMessageBTN = document.querySelector("#receieve-message");
 var messageBox = document.querySelector("#message-box");
+var clearMessageBTN = document.querySelector("#clear-btn");
 
 
 revieveMessageBTN.addEventListener('click', displaySelectedMessage);
+// clearMessageBTN.addEventListener('click', removeDisplayedMessage);
+
+
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length)
 }
 
+function checkButtonSelected() {
+     if (affirmationBTN.checked) {
+          return affirmations[getRandomIndex(affirmations)];
+     } else if (mantraBTN.checked) {
+          return mantras[getRandomIndex(mantras)];
+     }
+}
+
 function displaySelectedMessage() {
-    if (affirmationBTN.checked) {
-         messageBox.innerText = affirmations[getRandomIndex(affirmations)];
-    } else if (mantraBTN.checked) { 
-         messageBox.innerText = mantras[getRandomIndex(mantras)];
-    }
-};
\ No newline at end of file
+     var message = checkButtonSelected()
+     if (affirmationBTN.checked) {
+     messageBox.innerText = affirmations[getRandomIndex(affirmations)];
+     } else if (mantraBTN.checked) { 
+     messageBox.innerText = mantras[getRandomIndex(mantras)];
+     }
+     clearMessageBTN.classList.remove('hidden');
+};
```

Comment:
It looks like right now, if I don't select mantra or affirmation and click `Receive Message`, the clear button shows up:
<img width="910" alt="Screen Shot 2021-09-11 at 9 23 24 PM" src="https://user-images.githubusercontent.com/22228609/132970942-d92afc9c-4b99-48e8-9b29-7cad1483b22f.png">
To avoid this, you'll probably need a conditional somewhere in the `displaySelectedMessage` function telling it not to do anything if affirmation or mantra is not selected. Or alternatively you could disable the Receive Message button until one of those options is selected.